lib/experimental/disable-tinymce.php

```diff
@@ -61,14 +61,15 @@ function gutenberg_post_being_edited_requires_classic_block() {
 		return false;
 	}
 
-	// Handle the post editor.
-	if ( ! empty( $_GET['post'] ) && ! empty( $_GET['action'] ) && 'edit' === $_GET['action'] ) {
-		$current_post = get_post( intval( $_GET['post'] ) );
-		if ( ! $current_post || is_wp_error( $current_post ) ) {
-			return false;
-		}
+	// Continue only if we're in the post editor.
+	if ( empty( $_GET['post'] ) || empty( $_GET['action'] ) || 'edit' !== $_GET['action'] ) {
+		return false;
+	}
 
-		$content = $current_post->post_content;
+	// Bail if for some reason the post isn't found.
+	$current_post = get_post( intval( $_GET['post'] ) );
+	if ( ! $current_post || is_wp_error( $current_post ) ) {
```

Comment:
We can remove the `is_wp_error` check here. See https://developer.wordpress.org/reference/functions/get_post/#return

Reply:
I was almost sure that `WP_Post` could return a `WP_Error`, but I must have misremembered! Anyway, you're right, fixed in https://github.com/WordPress/gutenberg/pull/52935/commits/6bb68ee8969cc7339685407fde5ff99d6356ff51